tgui/packages/tgui/components/TextArea.js

```diff
@@ -104,7 +130,9 @@ export class TextArea extends Component {
     if (input) {
       input.value = toInputValue(nextValue);
     }
-
+    if (this.props.setInnerRef) {
+      this.props.setInnerRef(this.textareaRef);
+    }
```

Comment:
Not sure if I actually suggested you to do this, did you try doing `this.textareaRef = props.innerRef` and passing it a ref?